contrib/excise_stdlib.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Uses https://github.com/newren/git-filter-repo
+# Recommended use of `Github cli`
```

Comment:
Could you maybe add a comment pointing to the homepage (https://cli.github.com/), so that users can get to the installation instructions easily?

Reply:
`gh` is just a recommendation. It's not actively used in the script. I decided that the push to github is best done manually after the user inspects the result.